server/helper.ts

```diff
@@ -304,37 +328,109 @@ function getSearchkitRouter() {
     const fullUrl = `${host}/${req.body.index || "cmmstudy_en"}${req.url}`;
     logger.debug("Start Elasticsearch Request: %s", fullUrl);
 
+    const userQuery = req.body?.[0]?.params?.query || "";
+
     try {
       const response = await apiClient.handleRequest(req.body, {
         hooks: {
           // Hook to edit search request before it is executed
           beforeSearch: async (searchRequests) => {
             return searchRequests.map((sr) => {
+              const customQuery = userQuery
+                ? {
+                  query_string: {
+                    query: userQuery,
+                    default_operator: "AND" as QueryDslOperator,
+                    fields: SEARCH_FIELDS_WITH_BOOSTS
+                  }
+                }
+                : { match_all: {} };
+
               return {
                 ...sr,
                 body: {
                   ...sr.body,
-                  // Remove limit of 10000 results at the cost of speed
-                  track_total_hits: true
+                  track_total_hits: true, // Remove limit of 10000 results at the cost of speed
+                  query: {
+                    bool: {
+                      must: customQuery,
+                      filter: sr.body?.query?.bool?.filter || [] // Preserve existing filters
+                    }
+                  }
                 }
-              }
-            })
+              };
+            });
           }
         }
       });
-      res.send(response);
+
       logger.debug("Finished Elasticsearch Request to %s", fullUrl);
+      res.send(response);
     } catch (e: unknown) {
-      // When a connection error occurs send a 502 error to the client.
-      logger.error("Elasticsearch Request failed: %s: %s", fullUrl, e);
-      res.sendStatus(502);
+      logger.debug("Elasticsearch Request failed: %s", e);
+      const errorReason = extractElasticsearchErrorReason(e);
+      res.status(200).send({
+        results: [
+          {
+            hits: [],
+            nbHits: 0,
+            page: 0,
+            nbPages: 1,
+            hitsPerPage: 20,
+            processingTimeMS: 1,
+            query: userQuery,
+            params: '',
+            error: errorReason
+          }
+        ]
+      });
```

Comment:
```suggestion
      logger.error("Elasticsearch Request failed: %s: %s", fullUrl, e);
      res.sendStatus(502);
```

This shouldn't be necessary with `simple_query_string`